pallets/ocex/rpc/src/lib.rs

```diff
@@ -157,18 +158,27 @@ where
 		if !offchain_storage.acquire_offchain_lock(3).await {
 			return Err(runtime_error_into_rpc_err("Failed to acquire offchain lock"))
 		}
-		let runtime_api_result =
-			api.calculate_inventory_deviation(at).map_err(runtime_error_into_rpc_err)?;
-		let json =
-			serde_json::to_string(&runtime_api_result).map_err(runtime_error_into_rpc_err)?;
+		log::info!(target:"ocex","calculating the inventory deviation..");
+		let deviation =
+			match api.calculate_inventory_deviation(at).map_err(runtime_error_into_rpc_err)? {
+				Err(err) => {
+					log::error!(target:"ocex","Error calling calculate_inventory_deviation: {:?}",err);
+					return Err(runtime_error_into_rpc_err(
+						"Error calling calculate_inventory_deviation ",
+					))
+				},
+				Ok(deviation_map) => DeviationMap::new(deviation_map),
+			};
+		log::info!(target:"ocex","serializing the deviation map..");
+		let json = serde_json::to_string(&deviation).map_err(runtime_error_into_rpc_err)?;
 		Ok(json)
 	}
 
 	async fn fetch_checkpoint(
 		&self,
 		at: Option<<Block as BlockT>::Hash>,
 	) -> RpcResult<ObCheckpoint> {
-		self.deny_unsafe.check_if_safe()?;
+		//self.deny_unsafe.check_if_safe()?;
```

Comment:
Why is this commented out? at least, please add some inline comments why it is commented out and not just removed.